Protos/upstream/grpc/reflection/v1alpha/reflection.proto

```diff
@@ -11,13 +11,22 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 // Service exported by server reflection
 
+
+// Warning: this entire file is deprecated. Use this instead:
+// https://github.com/grpc/grpc-proto/blob/master/grpc/reflection/v1/reflection.proto
```

Comment:
Should we still be using this file (and all others that have been updated with this warning) at all?

Reply:
Yes, we offer both v1 and v1alpha, a number of tools using the reflection service only started using v1 recently. We also need to keep this for API compatibility reasons (our reflection service implementation can be configured to use  v1 or v1alpha).